data/blog/electron-12-0.md

```diff
@@ -0,0 +1,110 @@
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `89`
+    * [New in Chrome 88](https://developer.chrome.com/blog/new-in-chrome-88/)
+    * [New in Chrome 89](https://developer.chrome.com/blog/new-in-chrome-89/)
+* Node.js `14.16`
+    * [Node 14.16.0 blog post](https://nodejs.org/en/blog/release/v14.16.0/)
+    * [Node 14.0.0 blog post](https://nodejs.org/en/blog/release/v14.0.0/)
+* V8 `8.9`
+    * [V8 8.8 blog post](https://v8.dev/blog/v8-release-88)
+    * [V8 8.9 blog post](https://v8.dev/blog/v8-release-89)
+
+### Highlight Features
+
+* The ContextBridge `exposeInMainWorld` method can now expose non-object APIs. [#26834](https://github.com/electron/electron/pull/26834)
+* Upgraded from Node 12 to Node 14. [#23249](https://github.com/electron/electron/pull/25249)
+* Added a new `webFrameMain` API for accessing sub-frames of a `WebContents` instance from the main process. [#25464](https://github.com/electron/electron/pull/25464)
+
+See the [12.0.0 release notes](https://github.com/electron/electron/releases/tag/v12.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+* Deprecated the `remote` module. It is replaced by [`@electron/remote`](https://github.com/electron/remote). [#25293](https://github.com/electron/electron/pull/25293)
+    * If you are currently using the `remote` module, we've written [a guide to migrating to `@electron/remote` here.](https://github.com/electron/remote#migrating-from-remote)
+* Removed the ability to disable render process reuse in Electron 12. [Native Node modules](https://nodejs.org/api/addons.html) loaded in the renderer process must be either [N-API](https://nodejs.org/api/n-api.html) or [Context Aware](https://nodejs.org/api/addons.html#addons_context_aware_addons). Enforcing this change allows for stronger security, faster performance, and reduced maintenance workload. Read [this issue](https://github.com/electron/electron/issues/18397) for full details.
+* Changed the default of `crashReporter.start({ compress })` from `false` to `true`. [#25288](https://github.com/electron/electron/pull/25288) 
+* Changed the default value of `worldSafeExecuteJavaScript` to be `true`. [#27502](https://github.com/electron/electron/pull/27502) 
+
+More information about these and future changes can be found on the [Planned Breaking Changes](https://github.com/electron/electron/blob/master/docs/breaking-changes.md) page.
+
+## API Changes
+
+* Added `webFrameMain` API: The `webFrameMain` module can be used to look up frames across existing [`WebContents`](web-contents.md) instances. Navigation events are the common use case. More information about this new API can be found [here](https://github.com/electron/electron/pull/25464), and in our [documentation](https://www.electronjs.org/docs/api/web-frame-main).
```

Comment:
nit: maybe mention the fact that this is the main process equivalent of the existing `webFrame` API